samples/XAMLCatalog/XAMLCatalog/UIButtonViewController.m

```diff
@@ -278,8 +278,26 @@ - (UITableViewCell*)tableView:(UITableView*)tableView cellForRowAtIndexPath:(NSI
 
         [cell addSubview:label];
         [cell addSubview:button];
+    } else if (indexPath.row == 14) {
+        UILabel* label = [[UILabel alloc] initWithFrame:CGRectMake(_marginLeft, _marginTop, _labelWidth, _labelHeight)];
+        label.text = @"UIButton, Selected=true, Title=Blue";
+        
+        UIButton* button =
+        [[UIButton alloc] initWithFrame:CGRectMake(_marginLeft, _marginTop + _labelHeight, _defaultWidth, _defaultHeight)];
+        button.selected = true;
+        [button setTitleColor:[UIColor blueColor] forState:UIControlStateSelected];
+        [button setTitleColor:[UIColor blueColor] forState:UIControlStateNormal];
+        
+        [button setBackgroundImage:[UIImage imageNamed:@"blue_background.jpg"] forState:UIControlStateSelected];
+        [button setBackgroundImage:[UIImage imageNamed:@"yellow_background.jpg"] forState:UIControlStateNormal];
+        
+        [button setTitle:@"Selected Button" forState:UIControlStateSelected];
+        [button setTitle:@"Selected Button" forState:UIControlStateNormal];
```

Comment:
>[button setTitle:@"Selected Button" forState:UIControlStateNormal]; [](start = 8, length = 67)

This seems redundant to set the title to the same thing for two different states. The documentation shows that you only need to set the state for UIControlStateNormal and it will propagate to the other states.